README.md

```diff
@@ -72,10 +86,13 @@ In this file:
     The argument to the test stimulus block (`c` in this case) is a handle to the module under test.
 5.  In the test body, use `poke`, `step`, and `expect` operations to write the test:
     ```scala
-    c.in.poke(0.U)
-    c.out.expect(0.U)
-    c.in.poke(42.U)
-    c.out.expect(42.U)
+    c.io.in.poke(0.U)
```

Comment:
Once we add the register this needs to be changed to:

```.scala
    c.io.in.poke(0.U)
    c.clock.step()
    c.io.out.expect(0.U)
    c.io.in.poke(42.U)
    c.clock.step()
    c.io.out.expect(42.U)
```

Note that the canonical version for a single step is: `.step()` and not `.step(1)`.

Reply:
> Note that the canonical version for a single step is: `.step()` and not `.step(1)`.

This is typically something that should be included in the documentation :)